### PR TITLE
 UIREC-66 POL POL details displaying under Title information accordion

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
- Title information accordion is closed by default
- POL details' accordion
- Title appears above the title information accordion
- When the related purchase order has a workflow status of closed - an alert appears at top of title with message - "Purchase order is closed - [reason for closure]"

https://issues.folio.org/browse/UIREC-66

## Screenshot
image

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/TitleDetails/POLDetails/POLDetails.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import '@folio/stripes-acq-components/test/jest/__mock__';
+
+import POLDetails from './POLDetails';
+
+const renderPOLDetails = (polProp) => (render(
+  <IntlProvider locale="en">
+    <MemoryRouter>
+      <POLDetails
+        poLineId={polProp.poLineId}
```
Comment (aliaksei-chumakou) [inline src/TitleDetails/POLDetails/POLDetails.test.js]: it looks like you  have already props object below, `{...polDetails}`

File: src/TitleDetails/TitleDetails.js [hunk 2]
```diff
@@ -156,38 +162,63 @@ const TitleDetails = ({
       lastMenu={lastMenu}
     >
       <Row end="xs">
-        <Col xs={12}>
+        <Col xs={10}>
+          <MessageBanner
+            show={isOrderClosed}
```
Comment (aliaksei-chumakou) [inline src/TitleDetails/TitleDetails.js]: I don't think we need to use this technique. We use `{isOrderClosed && <MessageBanner show />}` to render element only when it's needed

File: test/bigtest/network/factories/order.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Factory } from 'miragejs';
+import faker from 'faker';
+
+import {
+  ORDER_STATUSES,
+} from '@folio/stripes-acq-components';
+
+export default Factory.extend({
```
Comment (aliaksei-chumakou) [inline test/bigtest/network/factories/order.js]: I don't think it's the best idea to copy-paste order models and factories round repos, maybe it makes sense to move them under stripes-acq-components

Comment (AndreiShumski2) [reply test/bigtest/network/factories/order.js]: moved config, model, serializer for orders to `stripes-acq-components` - [PR](https://github.com/folio-org/stripes-acq-components/pull/220)

**Pull request merged**